tpl/tplimpl/embedded/templates/helpers/plain_description_or_summary.html

```diff
@@ -0,0 +1,11 @@
+{{- with .Description | $.RenderString | plainify -}}
+{{- . -}}
+{{- else -}}
+{{- if .IsPage -}}
+{{- .Summary | plainify -}}
+{{- else -}}
+{{- with .Site.Params.description | $.RenderString | plainify -}}
+{{- . -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
A few comments:

- You don't need the leading `-` to strip whitespace
-  Use Indentation to make it easier to read.
- Avoid using `.RenderString` in this PR as, IMHO, that would be a breaking change, and it's not really addressing the main issue (#9818)
- Consider naming this file simply `page_description.html` (without create a new `helpers` folder)

So, something like this:

```suggestion
{{ with .Description | plainify -}}
  {{ . -}}
{{ else -}}
  {{ if .IsPage -}}
    {{ .Summary | plainify -}}
  {{ else -}}
    {{ with .Site.Params.description | plainify -}}
      {{ . -}}
    {{ end -}}
  {{ end -}}
{{ end -}}

```

Reply:
You’re right about the leading whitespace.

I followed the other templates in the vicinity and they had no indentation, so I’m gonna leave it as it is. But I like the idea and usually do indent template statements.

The `.RenderString` is necessary to translate Markdown into HTML. The following `plainify` only strips HTML and leaves Markdown formatting characters untouched. When you’re mentioned the "breaking change", I guess you had older Hugo versions in mind, which did not have `.RenderString`. So I could change that into an `markdownify`, which has been around much longer. But then again, I read somewhere, that ’.RenderString’ is more suitable than ’markdownify’. I don’t know, which aspect is more important.

I created the ’helpers’ folder, because the template is not meant to be called directly, but only within already existing templates. 

Maybe someone else could also take a look at this. I’m willing to resubmit the PR, if it helps.

Reply:
The `-` at the beginning of the templates are removed, `chomp` added to the `.Summary` pipeline.